src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/ImportResourceContainer.kt

```diff
@@ -114,7 +168,7 @@ class ImportResourceContainer(
                     categoryNode
                 }
             }
-            val projectResult = projectReader.constructProjectTree(container, project)
+            val projectResult = projectReader.constructProjectTree(container, project, zipEntryTreeBuilder)
```

Comment:
It feels like the zip format has permeated too many source files.  I think a better abstraction layer may be needed.

Reply:
Deferred until later refactoring.